samples/bluetooth/hci_ipc/nrf5340_cpunet_cis-bt_ll_sw_split.conf

```diff
@@ -5,7 +5,7 @@ CONFIG_ISR_STACK_SIZE=1024
 CONFIG_IDLE_STACK_SIZE=256
 CONFIG_MAIN_STACK_SIZE=512
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
-CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_HEAP_MEM_POOL_SIZE=6144
```

Comment:
Do we need this reduction for all ISO .confs? 

Isn't it only the combined one? 
Or is this just done to be consistent for all of them?